strings/en_US/messages.po

```diff
@@ -513,6 +513,39 @@ msgstr "Media (newspaper/radio)"
 msgid "Other"
 msgstr "Other"
 
+msgid "Raspberry Pi Foundation / CoderDojo Foundation outreach"
+msgstr "Raspberry Pi Foundation / CoderDojo Foundation outreach"
+
+msgid "Social media"
+msgstr "Social media"
+
+msgid "My employer (CSR scheme at work)"
+msgstr "My employer (CSR scheme at work)"
+
+msgid "Event / Conference / Training"
+msgstr "Event / Conference / Training"
+
+msgid "Future Learn"
+msgstr "Future Learn"
+
+msgid "Library or youth group"
+msgstr "Library or youth group"
+
+msgid "Other (comment field)"
+msgstr "Other (comment field)"
+
+msgid "Other Coding Organisations (no longer used)"
```

Comment:
Is there a reason why you didn't modify the existing one ? line 492

Reply:
No I probably should have with that one. 
Will amend